libqtile/widget/imapwidget.py

```diff
@@ -61,37 +61,48 @@ class ImapWidget(base.ThreadPoolText):
     ``mbox="~/Maildir/HomeMail/fred"``.  Note the nested sets of quotes! Labels
     can be whatever you choose, of course.
 
+    If you want you can give password directly then Gnome Keyring will not be
+    used.
+
     Widget requirements: keyring_.
 
     .. _keyring: https://pypi.org/project/keyring/
     """
 
     defaults = [
         ("mbox", '"INBOX"', "mailbox to fetch"),
-        ("label", "INBOX", "label for display"),
+        ("label", "INBOX: ", "label for display"),
```

Comment:
This seems unnecessary?

Reply:
the `text` was cosntructed with hardcoded `:`. I removed that, so wanted to get same functionality

Reply:
I see, ok. seems like the tests  need some fixing too.

Reply:
I'm trying to understand them. Not good at writing tests with python

Reply:
@tych0 it looks like I fixed tests. But docs are failing. Should them be fixed?

Reply:
yeah, everything needs to build correctly.